cppForSwig/BtcUtils.h

```diff
@@ -766,11 +775,48 @@ class BtcUtils
       return (8 + viLen + scrLen);
    }
 
+   static uint32_t TxWitnessCalcLength(uint8_t const * ptr)
```

Comment:
will deprectae calclength methods without size arg, make sure to avoid those as you progress forward